src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Runtime.cs

```diff
@@ -449,5 +539,91 @@ public static Uri CreateUri(string uri)
         {
             return new Uri(uri);
         }
+
+        public static bool SafeHandleAddRef(SafeHandle safeHandle)
```

Comment:
This does the same thing as `SafeHandle.DangerousAddRef` method. We would not ever add a public API like this.

Reply:
Definitely understand that but if we make it internal or private then the methods get linked out.  Is there a common way to make private or internal methods that are not referenced in the assembly not to be linked out?